src/Idris/REPLOpts.idr

```diff
@@ -46,7 +46,7 @@ export
 defaultOpts : Maybe String -> OutputMode -> List (String, Codegen) -> REPLOpts
 defaultOpts fname outmode cgs
     = MkREPLOpts False NormaliseAll fname (litStyle fname) "" "vim"
```

Comment:
Now that we have named applies, it may be worth refactoring this unreadable
`MkREPLOpts` application to:

```idris
MkREPLOpts
  { showTypes = ...
  , evalMode  = ...
  , ...
  }
```

? :)

Reply:
Sure! This way is definitely more readable. By the way, is this the change that is mentioned in the changelog as "Added new syntax for named applications of explicit arguments"? I read that and had no idea what it was about 😅. `f {x [= t], x [= t], ...}` is pretty cryptic though.